<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_ed_plane</class>
 <widget class="QDialog" name="Dialog_ed_plane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>1041</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>航班号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_fn">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>起飞地</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_tc">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>到达地</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_lc">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>价格</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_p">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>折扣</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_d">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>起飞时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_tt">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>到达时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_at">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>44</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder></zorder>
  <zorder>pushButton</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>horizontalSpacer_2</zorder>
  <zorder>horizontalSpacer_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
